packages/shared/src/write-archive/archive-file.ts

```diff
@@ -12,6 +12,8 @@ const RESERVED_PATHNAMES: string[] = [
   '/index.json',
   '/iframe.html',
   '/index.html',
+  '/runtime~main.iframe.bundle.js',
+  '/main.iframe.bundle.js',
```

Comment:
@tmeasday @vanessayuenn @shilman When we run the SB build command, what file names are always consistent? We will want to include these here.

Reply:
I think there's a solution that would fix this without needing to specify every file that could possibly be overwritten. I'll try to work on that today.